collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1492,7 +1492,7 @@ public void showView(SwipeHandler.View next, AnimationType from) {
         // adjust which view is in the layout container...
         SwipeHandler.View staleView = currentView;
         currentView = next;
-        swipeHandler.setView(currentView);
+//        swipeHandler.setView(currentView);
```

Comment:
This should be reverted I guess.

Reply:
Ach I forgot to redo `git add` after taking this out again.